src/content/docs/style-guide/writing-docs/processes-procedures/create-preview-document.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Create content for the preview docs site
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+redirects:
+---
+
+The preview docs site hosts preview docs for upcoming features that we share with customers.
+
+Unlike the public docs site, the preview docs site is not publicly editable and does not appear in search engine results. Additionally, the site is not indexed, so you can't navigate from one preview doc to another. Because the preview docs site is located in a different repo than the public docs site, contributing looks a little different.
+
+Follow the steps below to create and edit preview docs.
+
+## Set up your local environment [#do-first]
+
+1. Go to the [`beta-docs-site repo`](https://github.com/newrelic/beta-docs-site). If you do not have access, contact the #doc-eng-help Slack channel for help.
+2. Clone the beta-docs-site repo on your computer.
+3. Make sure you're working in the correct repo on Github Desktop.
+4. Create a branch off the `main` branch.
+  
+## Create a new preview doc [#create-docs]
+
+Once you're in your working branch:
+
+1. Using your text editor, create a new `.mdx` file or copy an existing preview doc for the frontmatter template.
+2. Save the file in `/src/content/docs/`.
+3. Write your content, making sure to include any info about accessing the preview feature you're documenting.
+4. Commit your changes and create a pull request.
+5. Share the preview link with reviewers and implement any feedback.
+6. Merge the final PR.
+7. Create the official link by pasting the file path name after `https://docs-preview.newrelic.com/docs/` (For example: `https://docs-preview.newrelic.com/docs/new-feature-name`)
+8. Provide reviewers the official link to share with customers.
+
+## Optional boiler plate text [#boiler-plate]
```

Comment:
It might be worth clarifying why this is optional and when someone might choose to add this callout or not.